ADLStream/adlstream.py

```diff
@@ -253,6 +260,9 @@ def __init__(
         self.train_gpu_index = train_gpu_index
         self.predict_gpu_index = predict_gpu_index
         self.log_file = log_file
+        self.X_shape = None
```

Comment:
Minor change - change the name of the variable, it should not be capitalized: `X_shape` --> `x_shape`